packages/core/src/Tab/Tab.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import * as Tabs from '@radix-ui/react-tabs'
+import { Text } from '../Text'
+import { TabContainer, TabList, TabTriggerChip, TabTriggerButton, TabTriggerRadio } from './Tab.styled'
+import { Box } from '../Box'
+import { Radio } from '../Radio'
+import { Label } from '../Label'
+
+export interface ITabsProps {
+  orientation?: 'horizontal' | 'vertical'
+  onHover?: boolean
+  onClick?: () => void
+  size?: string
+  tabsContent: {
+    id: string
+    children: React.ReactNode
+  }[]
+  tabsData: {
+    id: string
+    icon?: React.ReactNode
+    text: string
+    onClick?: () => void
+  }[]
+  variant?: string
+  type?: 'chip' | 'radio' | 'button'
+}
+
+const PclnTab = ({
+  tabsContent,
+  tabsData,
+  orientation = 'horizontal',
+  onClick,
+  onHover = true,
+  size = 'md',
+  type = 'button',
+}: ITabsProps) => {
+  const [isActive, setIsActive] = React.useState(() => {
+    const initialActiveState = new Array(tabsData.length).fill(false)
+    initialActiveState[0] = true
+    return initialActiveState
+  })
+  const tabLink = tabsContent[0].id
+  return (
+    <TabContainer type={type} orientation={orientation} defaultValue={tabLink}>
+      <TabList orientation={orientation} aria-label='Pcln Tabs'>
+        {tabsData.map((tab, index) => {
+          return (
+            <>
+              {type === 'chip' ? (
+                <TabTriggerChip
```

Comment:
I think we should move the `chip` and `button` trigger components to separate files to keep `Tab` tidy